.github/CONTRIBUTING.rst

```diff
@@ -66,6 +66,12 @@ Here's how to make a one-off code change.
    - You can refer to relevant issues in the commit message by writing, e.g., "#105".
 
    - Your code should adhere to the `PEP 8 Style Guide`_, with the exception that we have a maximum line length of 99.
+   
+   - Document your code. This project uses `sphinx`_ to generate static HTML docs. To build them, use the ``Makefile`` in the ``docs`` directory:
```

Comment:
Instead of `use the ``Makefile``... `, maybe just say `run … from within your PTB directory`? Else it reads like one should `cd docs` before running the snippet

Reply:
Yes, that was confusing. I've amended the commit!